Tchap/Modules/Avatar/AvatarImageViewModel.swift

```diff
@@ -19,12 +19,28 @@ import Foundation
 @objcMembers
 final class AvatarImageViewModel: NSObject {
     
-    let thumbStringUrl: String?
+    let avatarContentURI: String?
+    let mediaManager: MXMediaManager?
```

Comment:
We should find a way to not set the MXMediaManager on the view model in the future
.

Reply:
agree